stackstorm-nexus3/README.md

```diff
@@ -0,0 +1,80 @@
+# Stackstorm-Sontatype Nexus3
+
+
+Sontatype nexus3 stackstorm pack
+
+## Installation
+
+
+Install this pack with: `st2 pack install file:///$PWD`
+
+Or if in remote repository: `st2 pack install https://github.com/MY/PACK`
+
+## Configuration
+
+Copy the example configuration in [nexus3.yaml.example](./nexus3.yaml.example)
+to `/opt/stackstorm/configs/nexus3.yaml` and edit as required.
+
+add nexus3 server connection profile:
+
+* ``url`` - URL of the nexus3 server (e.g. ``http://localhost:8081``)
+* ``user`` - username
+* ``password`` - Password
+* ``verify`` - https tls verify
```

Comment:
I would add a note that this is only used if the `url` option specifies an `https` connection.